invenio_cli/helpers/scripts.py

```diff
@@ -285,14 +279,14 @@ def signal_handler(sig, frame):
     server.wait()
 
 
-def _server_prod():
-    pass
-
-
 def server(dev=True, docker_helper=None, app_name='invenio-rdm',
            verbose=False, loglevel=logging.WARN, logfile='invenio-cli.log'):
     """Run server."""
     if dev:
         _server_dev(docker_helper, loglevel, logfile, verbose)
     else:
-        _server_prod()
+        click.secho("Starting docker containers. " +
+                    "It might take up to a minute.", fg="green")
+        click.secho("Use --stop to stop server.", fg="green")
+        docker_helper.start_containers()
+        time.sleep(60)
```

Comment:
Is this needed because otherwise the python (parent) script finishes executing before the child is ready and therefore the child gets cancelled?

Reply:
Nope, this is because the docker contianers take a bit to be up, and I want to avoid the user going to the browser and seeing a "site not reachable" error just because it was not patient enough :D